.github/workflows/docker.yml

```diff
@@ -19,8 +19,8 @@ jobs:
     - name: Build
       run: |-
         docker build \
-          -t us-west2-docker.pkg.dev/janus-artifacts/divviup-ts/divviup_ts_interop_client:dap-draft-01 \
-          -t us-west2-docker.pkg.dev/divviup-artifacts-public/divviup-ts/divviup_ts_interop_client:dap-draft-01 \
+          -t us-west2-docker.pkg.dev/janus-artifacts/divviup-ts/divviup_ts_interop_client:dap-draft-02 \
+          -t us-west2-docker.pkg.dev/divviup-artifacts-public/divviup-ts/divviup_ts_interop_client:dap-draft-02 \
```

Comment:
This is somewhat orthogonal to this change, but I think we should tag these images with the git commit they were built from so that if we're later doing integration tests against Janus or some other DAP server, then we can figure out what divviup-ts was in use.